src/store/assets.ts

```diff
@@ -0,0 +1,24 @@
+import { create } from 'zustand'
+
+import { initialAssetsById } from '../constants/assets'
+import type { Asset, AssetId } from '../types/assets'
+
+interface AssetsState {
+  byId: Record<AssetId, Asset>
```

Comment:
Note not a partial, for the time being, we're working with hardcoded constant so any asset we lookup *will* be defined